### PR TITLE
Mach learn comp clust local auth disabled

### PR DESCRIPTION
**Pull request by c0mmad0r3**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

Custom Policy ID: CKV_AZURE_142
Custom Policy name: Ensure Machine Learning Compute Cluster Local Authentication is disabled
Custom Policy IaC type: Terraform
Custom Policy type and provider: AzureRM
IaC configuration documentation (If available):  https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/machine_learning_compute_cluster#local_auth_enabled
Sample Terraform configuration file:  N/A
Any additional information that would help other members to better understand the check: https://docs.microsoft.com/en-us/azure/machine-learning/how-to-setup-authentication & https://docs.microsoft.com/en-us/azure/machine-learning/how-to-create-attach-compute-cluster

### CODE REVIEW

File: checkov/terraform/checks/resource/azure/CosmosDBLocalAuthDisabled.py [hunk 1]
```diff
@@ -19,7 +19,10 @@ def __init__(self):
 
     def get_inspected_key(self) -> str:
         return "local_authentication_disabled"
+<<<<<<< HEAD
+=======
         
+>>>>>>> cdc760b2a1bba75f79182664a611db240143e0d6
```
Comment (gruebel) [inline checkov/terraform/checks/resource/azure/CosmosDBLocalAuthDisabled.py:25]: this looks like leftovers from a merge conflict 😄  please remove them

**Review by gruebel — COMMENTED** [review]
other than that looks good 👍

**Comment by c0mmad0r3** [comment]
@gruebel 😄 my bad.  Removed.

**Review by gruebel — APPROVED** [review]
good job 🥇

File: tests/terraform/checks/resource/azure/test_MLCCLocalAuthDisabled.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import os
+import unittest
+
+from checkov.runner_filter import RunnerFilter
+from checkov.terraform.runner import Runner
+from MLCCLoAuthDisabled import check
```
Comment (gruebel) [inline tests/terraform/checks/resource/azure/test_MLCCLocalAuthDisabled.py:6]:
```suggestion
from checkov.terraform.checks.resource.azure.MLCCLoAuthDisabled import check
```
I saw a test was failing, I think that is the error 🙂

**Pull request closed without merging**